src/support_sphere/lib/utils/constants.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+
+/// Environment variables and shared app constants.
+abstract class Constants {
+  static const String supabaseUrl = String.fromEnvironment(
+    'SUPABASE_URL',
+    defaultValue: 'http://localhost',
+  );
+
+  static const String supabaseAnnonKey = String.fromEnvironment(
+    'SUPABASE_ANNON_KEY',
```

Comment:
nit: `annon` -> `anon`

```suggestion
  static const String supabaseAnonKey = String.fromEnvironment(
    'SUPABASE_ANON_KEY',
```